post-compute/src/compute/encryption.rs

```diff
@@ -0,0 +1,947 @@
+use crate::compute::errors::ReplicateStatusCause;
+use crate::compute::web2_result::{Web2ResultInterface, Web2ResultService};
+use aes::{
+    Aes256,
+    cipher::{BlockEncryptMut, KeyIvInit, block_padding::Pkcs7},
+};
+use cbc::Encryptor;
+use log::error;
+use rand::{RngCore, rngs::OsRng};
+use rsa::{Pkcs1v15Encrypt, RsaPublicKey, pkcs8::DecodePublicKey};
+use sha3::{Digest, Sha3_256};
+use std::{fs, path::Path};
+
+/// 256-bit key (32 bytes)
+pub const AES_KEY_LENGTH: usize = 32;
+/// 128-bit IV (16 bytes) same as the AES block size
+pub const AES_IV_LENGTH: usize = 16;
```

Comment:
After examples cleanup, I think the `pub` modifier which was added to let examples pass is no more useful and can be removed to keep the same visibility than in `tee-worker-post-compute-rust`
```suggestion
/// 256-bit key (32 bytes)
const AES_KEY_LENGTH: usize = 32;
/// 128-bit IV (16 bytes) same as the AES block size
const AES_IV_LENGTH: usize = 16;
```